docs/components/Intro.js

```diff
@@ -1,346 +1,227 @@
-import React from "react";
+import React, { useState } from "react";
 import Link from "@docusaurus/Link";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 import { SdkLogos } from "./SdkLogos";
 
-function InfoSVG() {
-  return (
-    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path
-        d="M12 6.5C8.67188 6.5 6 9.19531 6 12.5C6 15.8281 8.67188 18.5 12 18.5C15.3047 18.5 18 15.8281 18 12.5C18 9.19531 15.3047 6.5 12 6.5ZM12 17.375C9.30469 17.375 7.125 15.1953 7.125 12.5C7.125 9.82812 9.30469 7.625 12 7.625C14.6719 7.625 16.875 9.82812 16.875 12.5C16.875 15.1953 14.6719 17.375 12 17.375ZM12.9375 14.375H12.5625V12.3125C12.5625 12.0078 12.3047 11.75 12 11.75H11.25C10.9219 11.75 10.6875 12.0078 10.6875 12.3125C10.6875 12.6406 10.9219 12.875 11.25 12.875H11.4375V14.375H11.0625C10.7344 14.375 10.5 14.6328 10.5 14.9375C10.5 15.2656 10.7344 15.5 11.0625 15.5H12.9375C13.2422 15.5 13.5 15.2656 13.5 14.9375C13.5 14.6328 13.2422 14.375 12.9375 14.375ZM12 11C12.3984 11 12.75 10.6719 12.75 10.25C12.75 9.85156 12.3984 9.5 12 9.5C11.5781 9.5 11.25 9.85156 11.25 10.25C11.25 10.6719 11.5781 11 12 11Z"
-        fill="currentColor"
-      />
-      <path
-        d="M12 6.5C8.67188 6.5 6 9.19531 6 12.5C6 15.8281 8.67188 18.5 12 18.5C15.3047 18.5 18 15.8281 18 12.5C18 9.19531 15.3047 6.5 12 6.5ZM12 17.375C9.30469 17.375 7.125 15.1953 7.125 12.5C7.125 9.82812 9.30469 7.625 12 7.625C14.6719 7.625 16.875 9.82812 16.875 12.5C16.875 15.1953 14.6719 17.375 12 17.375ZM12.9375 14.375H12.5625V12.3125C12.5625 12.0078 12.3047 11.75 12 11.75H11.25C10.9219 11.75 10.6875 12.0078 10.6875 12.3125C10.6875 12.6406 10.9219 12.875 11.25 12.875H11.4375V14.375H11.0625C10.7344 14.375 10.5 14.6328 10.5 14.9375C10.5 15.2656 10.7344 15.5 11.0625 15.5H12.9375C13.2422 15.5 13.5 15.2656 13.5 14.9375C13.5 14.6328 13.2422 14.375 12.9375 14.375ZM12 11C12.3984 11 12.75 10.6719 12.75 10.25C12.75 9.85156 12.3984 9.5 12 9.5C11.5781 9.5 11.25 9.85156 11.25 10.25C11.25 10.6719 11.5781 11 12 11Z"
-        fill="currentColor"
-      />
-    </svg>
-  );
-}
+const coreConcepts = [
+  {
+    path: "https://learn.temporal.io/getting_started#set-up-your-development-environment",
+    name: "Set up your development environment",
+  },
+  {
+    path: "https://learn.temporal.io/getting_started/#run-your-first-program",
+    name: "Run your first program",
+  },
+  {
+    path: "https://learn.temporal.io/getting_started/#run-your-first-program",
+    name: "Build an application from scratch",
+  },
+  {
+    path: "https://learn.temporal.io/courses/temporal_101/",
+    name: "Temporal 101 with Go, Java, TypeScript, Python",
+  },
+  {
+    path: "https://learn.temporal.io/courses/temporal_102/",
+    name: "Temporal 102 with Go",
+  },
+  {
+    path: "https://learn.temporal.io/courses/intro_to_temporal_cloud/",
+    name: "Introduction to Temporal Cloud",
+  },
```

Comment:
I think we are going to drop these and just have the prereqs, run your first app, and hello world from scratch -

@napcs correct?

Reply:
👍 dropped these